plugin/helpers_bots.go

```diff
@@ -54,6 +55,15 @@ func FilterUserIDs(filterUserIDs []string) ShouldProcessMessageOption {
 	}
 }
 
+// OnlyBotDMs configures a call to ShouldProcessMessage to return true only for post in Bot DM channel.
```

Comment:
```suggestion
// OnlyBotDMs configures a call to ShouldProcessMessage to return true only for direct messages sent to the bot created by EnsureBot.
```

Reply:
@lieut-data I don't seem to get what this DM channel is? Is this the same channel when I use github plugin? 
![image](https://user-images.githubusercontent.com/37879062/67614342-20b1d500-f7d9-11e9-80b6-5288ff8136bc.png)

Reply:
Yes, that's exactly it. Some plugins just want to "listen" to messages sent to their own bot, via a direct message.